test/interactions/panZoomInteractionTests.ts

```diff
@@ -181,6 +181,29 @@ describe("Interactions", () => {
       svg.remove();
     });
 
+    it("pinching inside one component does not affect another component", () => {
+      var svg2 = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+      var component2 = new Plottable.Component();
+      component2.renderTo(svg2);
```

Comment:
We don't need the component to be in a separate svg.  The original PanZoomInteraction would fail even if the components were in the same svg.  Let's just stick to 1 svg for simplicity.